project-koku/koku
# [COST-5400] - use amortized cost for ocp on all endpoints

## Jira Ticket

[COST-5400](https://issues.redhat.com/browse/COST-5400)

## Description

This change will switch OCP on ALL endpoints to use calculated amortised cost when inserting data from OCP on AWS lines

## Testing

1. Checkout Branch
2. Restart Koku
3. Load OCP on AWS data (this needs to include some saving plan costs)
    1. You can use the ocp_on_aws_source_raw_calc smoke test
4. Verify the following endpoints have he same costs:

http://localhost:8000/api/cost-management/v1/reports/openshift/infrastructures/aws/costs/?cost_type=calculated_amortized_cost
http://localhost:8000/api/cost-management/v1/reports/openshift/costs/
http://localhost:8000/api/cost-management/v1/reports/openshift/infrastructures/all/costs/

## Release Notes

```markdown
* [COST-COST-5400](https://issues.redhat.com/browse/COST-COST-5400) Switch OCP on ALL API endpoints to calculated amortised cost.
```
